### ISSUE TITLE
Autoselecting does not give clear feedback on JAWS

### ISSUE THREAD

**Issue by tvararu** [alphagov/accessible-autocomplete#57]
When using the `autoselect` option, when the user types in a query that brings up suggestions, the list of suggestions will update and visually highlight the first element, while offering audio feedback informing the user that it's selected. The user may press `enter` to confirm it.

I've tested this on JAWS17 and the typeahead does not give clear enough feedback after confirming that the user has successfully chosen the first option. Other screen reader + browser combinations accomplish this usually by simply reading out the new value; JAWS17 just says "enter," and on FireFox it also says "collapsed."

| Browser | JAWS17 |
|---|---|
| Chrome | Broken |
| FireFox | Broken |
| IE11 | Broken |

A possible solution is to blur and refocus the input field but a bit of investigation and testing is needed to fix this.

**Comment by tvararu** [comment]
I have tried the following solutions:

- *Doesn't work*: Blur the input and then focus the input. This will cause JAWS to start reading the document from the top of the page.
- *Doesn't work*: Focus the selected option and then focus the input. This does nothing.
- *Works*: Focus the selected option, wait 1000ms, focus the input. This is slightly jarring because it tells the user they have entered a listbox and then tells them they're on an input, but does the job, albeit, very poorly.

**Comment by tvararu** [comment]
Idea: worth investigating if other typeaheads have managed to solve this issue, and how.

**Comment by tvararu** [comment]
Closing as it's unlikely there's a good way to do this.
